docs/install_guides/troubleshooting-steps.md

```diff
@@ -23,8 +23,8 @@ Check the versions of OpenVINO Runtime and Development Tools
           .. code-block:: py
              :force:
 
-             from openvino.tools.mo import convert_model
-             ov_model = convert_model(version=True)
+             import openvino as ov
+             ov_model = ov.convert_model(version=True)
```

Comment:
will `ovc -h` print version?
or maybe even `ovc --version` ?

CC @popovaan

Reply:
ov.convert_model doesn't have `version` parameter. Use `ov.__version__`.

`ovc --version` gives output like this: `OpenVINO Model Converter (ovc) 2023.1.0-12028-dab01d286e6`

Reply:
Exactly when testing of snippets would help.